enterprise/frontend/src/embedding-sdk/hooks/private/use-load-static-question.ts

```diff
@@ -45,6 +34,7 @@ export function useLoadStaticQuestion(
 
   useEffect(() => {
     const cancelDeferred = defer();
+    let ignore = false; // flag to ignore the result if the component unmounts: https://react.dev/learn/you-might-not-need-an-effect#fetching-data
```

Comment:
This is also a lot simpler to understand imo than the old solution, it basically says: if we cleaned up the effect -> skip side effects